scripts/lint_jjb.py

```diff
@@ -225,6 +227,25 @@ def check_timed_trigger_value(schedule, data):
     return return_value
 
 
+# Checks the job for the proper repo_url protocol.
+# Returns True if job has a invalid repo_url protocol, otherwise returns False
+#
+def invalid_protocol(job, file_name):
+    error_message = ("{f}/{j}: Invalid repo_url protocol - "
+                     " only https protocol is allowed."
+                     " Expected value similar to "
+                     " https://github.com/..."
+                     " but received: {r}\n"
+                     .format(j=job['name'], f=file_name,
+                             r=job['repo_url']))
+
+    if job['repo_url'].startswith('https://'):
```

Comment:
Job's don't tend to have a repo_url key, the important key in this case is
```
job
  properties
    github
      url
```

eg:
```
- job:
    name: 'Component-Release-Trigger'
    project-type: pipeline
    concurrent: true
    triggers:
      - github-pull-request:
          org-list:
            - rcbops
          github-hooks: true
          trigger-phrase: ":shipit:|:shipit: skip validation"
          only-trigger-phrase: true
          auth-id: "github_account_rpc_jenkins_svc"
          status-context: 'release'
          cancel-builds-on-update: true
    properties:
      - github:
          url: "https://github.com/rcbops/releases"
```